DICOM/src/Microsoft.Health.DeID.SharedLib/Settings/EncryptSetting.cs

```diff
@@ -10,7 +10,7 @@
 
 namespace Microsoft.Health.DeID.SharedLib.Settings
 {
-    public class EncryptionSetting
+    public class EncryptSetting
```

Comment:
Instead of changing EncryptionSetting to EncryptSetting, I'd prefer changing the name of other settings from verb+setting to noun+setting
It's a soft suggesting though.